scraper.rb

```diff
@@ -9,33 +9,24 @@
 require 'open-uri/cached'
 OpenURI::Cache.cache_path = '.cache'
 
-BASE = 'http://www.parliament.gov.zm'
-
 def scrape(h)
   url, klass = h.to_a.first
   klass.new(response: Scraped::Request.new(url: url).response)
 end
 
-ScraperWiki.sqliteexecute('DELETE FROM data') rescue nil
-
-# We should really extract these from the 'Next' links
-pages = [
-  '/members-of-parliament',
-  '/members-of-parliament/page/1/0',
-  '/members-of-parliament/page/2/0',
-]
-
-added = 0
-pages.each do |page|
-  url = URI.join(BASE, page).to_s
-  warn "Fetching #{url}"
-  (scrape url => MembersPage).member_rows.each do |row|
+def data_for_members(url)
+  members_page = (scrape url => MembersPage)
+  data = members_page.member_rows.map do |row|
```

Comment:
I'm a little surprised that rubocop doesn't catch this, but you seem to have two separate `data` variables on the go simultaneously here, each meaning something different. The inner one doesn't seem to actually be required though.